typegraph/node/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@typegraph/sdk",
+  "version": "0.3.1",
+  "description": "Declarative API development platform. Build serverless backends with zero-trust and less code, no matter where and how your (legacy) systems are.",
+  "type": "module",
+  "main": "mod.js",
```

Comment:
can we put it inside the src folder? and rename index?

Reply:
`mod.ts` is already inside src folder. But sure we can rename it!